src/requestManager.h

```diff
@@ -160,6 +160,9 @@ class CRequestManager
     // in preparation for shutdown
     void Cleanup();
 
+    // Size of last block that was successfully connected at the tip.
+    std::atomic<uint64_t> nLastBlockSize;
+
```

Comment:
I feel like this should not be in the requestManager, since it doesn't have anything to do with handling tx or block requests.  What about using a global variable?

Reply:
ok done